client/admin/authorUtils/authorLookup/authorLookup.jsx

```diff
@@ -0,0 +1,90 @@
+import './authorLookup.less';
+
+import React from 'react';
+import request from 'superagent';
+
+const authorLookup = ()=>{
+	const [author, setAuthor] = React.useState('');
+	const [searching, setSearching] = React.useState(false);
+	const [results, setResults] = React.useState([]);
+
+	const lookup = async ()=>{
+		if(!author) return;
+
+		setSearching(true);
+		setResults([]);
+
+		await request.get(`/admin/user/list/${author}`)
+            .then((brews)=>{
+            	setResults(brews.body);
+            	setSearching(false);
+            });
+	};
```

Comment:
One of the benefits of `await` is we don't need to use the `.then` syntax, for better legibility.

```suggestion
		const brews = await request.get(`/admin/user/list/${author}`);
		setResults(brews.body);
		setSearching(false);
```